src/main/scala/cromwell/engine/SymbolStore.scala

```diff
@@ -26,9 +26,46 @@ class SymbolStore(binding: WdlBinding, inputs: Map[FullyQualifiedName, WdlValue]
     assignSymbolStoreEntry(fullyQualifiedName, value, input = true)
   }
 
+  binding.workflows foreach {workflow =>
+    workflow.calls foreach {call =>
+      call.inputMappings foreach {
+        case (k, v) => assignSymbolStoreEntry(s"${call.fullyQualifiedName}.$k", v, input = true)
+      }
+    }
+  }
+
   def locallyQualifiedInputs(call: Call): Map[String, WdlValue] = {
+
+    def lookup(s: String): WdlValue = {
+      var workflow = call.parent.getOrElse {
+        throw new WdlExpressionException("Expecting 'call' to have a 'workflow' parent")
+      }.asInstanceOf[Workflow]
+      val matchedCall = workflow.calls.find{c => c.name == s}.getOrElse {
+        throw new WdlExpressionException(s"Expecting to find a call with name '$s'")
+      }
+      val callOutputs = store.collect{ case entry if entry.scope == matchedCall.fullyQualifiedName && entry.isOutput =>
+          val value = entry.wdlValue match {
```

Comment:
Similarly, it'd be good to have this handled by a function so the collect itself is one line